### ISSUE TITLE
Elgato Key Light Mini Support

### ISSUE THREAD

**Issue by GaryGSC** [frenck/python-elgato#673]
Just wanted to let others know that this library works beautifully with the [Key Light Mini](https://www.elgato.com/en/key-light-mini).

I can control brightness and temperature. The library correctly identifies the minimum and maximum temperatures supported. The `Identify` service works, too.

The only thing that seems to be missing is the battery level, but since my lights are always plugged in, that didn't matter to me.

## Proposed changes

- List Key Light Mini as a supported device in the README (Can we consider it compatible and tested?)
- Get battery information

**Comment by frenck** [comment]
Nice :)

Wasn't aware of this device. Your issue made me order one, which I received today.

I've noticed some changes in general in newer Elgato firmware, that I can apply/add to this library and discovered some new API endpoints too.

For the mini, there is a lot to support/pull out of it. I've been able to get all battery and charging information and got into the battery-related settings as well (controlling bypass/studio mode, energy-saving behaviors/settings, and stuff like that).

Will extend the library with all the found information and controls.

../Frenck
